ui/src/app/components/apps-panel/expanded-app.tsx

```diff
@@ -269,19 +269,24 @@ export const ExpandedApp = (props: ExpandedAppProps) => {
           // TODO: support Cromwell + other User Apps
           appType === UIAppType.JUPYTER && <RuntimeCost />
         }
-        <Clickable
-          disabled={!trashEnabled}
-          style={
-            trashEnabled
-              ? styles.enabledTrashButton
-              : styles.disabledTrashButton
-          }
-          onClick={onClickDelete}
-          data-test-id={`${appType}-delete-button`}
-          propagateDataTestId
+        <TooltipTrigger
+          disabled={trashEnabled}
+          content='Your application must be running in order to be deleted.'
         >
-          <FontAwesomeIcon icon={faTrashCan} />
-        </Clickable>
+          <Clickable
+            disabled={!trashEnabled}
+            style={
+              trashEnabled
+                ? styles.enabledTrashButton
+                : styles.disabledTrashButton
+            }
+            onClick={onClickDelete}
+            data-test-id={`${appType}-delete-button`}
+            propagateDataTestId
```

Comment:
This is used by an E2E test. Otherwise, the `data-test-id` is lost after rendering to HTML.

At the moment, I work around data-test-id where I can. In my opinion, we'd ideally have no uses of data-test-id, but I haven't put much thought into alternatives.